### PR TITLE
fix(DB/gob): Remove Replace Repool Durotar Herbs

### PR DESCRIPTION
**Pull request by neifion-00000000**
Co-Authored with @quigiboo 
## Changes Proposed:
image
Player experience should match these images now, pools are as shown and max pool is determined by max of a single type of circles (two types of circles were used to get a sample of different herb placements to increase pooling accuracy)

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [X] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [X] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.
- [X] I have done some in game tests.  A light test of the pooling was done, and every single node placement was checked (and organized as shown in the image)

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.  .go ga and check incrementally:
2. 10601+ 16 Mageroyal
3. 21871+ 46 Earthroot
4. 64519 + 76 Peacebloom
5. 55775 + 68 Silverleaf

## Known Issues and TODO List:

- Oh this is done, like forever, I think.  The only improvement that can be done here is general herb/mine timers*, I set to lowest timers since we can not do dynamic timers

*Some pure psychopath can spend the averaged out 144hrs needed to solve every single pool here if they'd like to

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1696800156876281900.sql [hunk 1]
```diff
@@ -0,0 +1,570 @@
+--
+-- Remove existing Durotar Herbalism Nodes:
+DELETE FROM pool_gameobject WHERE pool_entry BETWEEN 498 AND 501;
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1696800156876281900.sql:3]:
```suggestion
DELETE FROM `pool_gameobject` WHERE `pool_entry` BETWEEN 498 AND 501;
```

File: data/sql/updates/pending_db_world/rev_1696800156876281900.sql [hunk 2]
```diff
@@ -0,0 +1,570 @@
...
+DELETE FROM pool_template WHERE entry BETWEEN 498 AND 501;
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1696800156876281900.sql:4]:
```suggestion
DELETE FROM `pool_template` WHERE `entry` BETWEEN 498 AND 501;
```

File: data/sql/updates/pending_db_world/rev_1696800156876281900.sql [hunk 3]
```diff
@@ -0,0 +1,570 @@
...
+(@OGUID+2, 1620, 1, 14, 0, 1, 1, 763.56890869140625, -5037.0380859375, 7.180908203125, 0.366517573595046997, 0, 0, 0.182234764099121093, 0.98325502872467041, 360, 255, 1, 46779, '3/16 all known Durotar Mageroyal sniffed'),
+(@OGUID+8, 1620, 1, 14, 0, 1, 1, 931.2001953125, -4038.233154296875, -13.346710205078125, 3.9793548583984375, 0, 0, -0.9135446548461914, 0.406738430261611938, 360, 255, 1, 46779, '9/16 all known Durotar Mageroyal sniffed'),
+(@OGUID+0, 1620, 1, 14, 0, 1, 1, 1450.59326171875, -4902.47802734375, 9.712756156921386718, 3.839725255966186523, 0, 0, -0.93969249725341796, 0.34202045202255249, 360, 255, 1, 46779, '1/16 all known Durotar Mageroyal sniffed'),
+(@OGUID+15, 1620, 1, 14, 0, 1, 1, 122.14996337890625, -4474.04931640625, 37.63466262817382812, 2.076939344406127929, 0, 0, 0.861628532409667968, 0.50753939151763916, 360, 255, 1, 46779, '16/16 all known Durotar Mageroyal sniffed');
+
+-- Make Mageroyal Pooling:
+SET @OBJECTPOOLS :=112;
+DELETE FROM `pool_gameobject` WHERE `guid` BETWEEN @OGUID+0 AND @OGUID+3;
+DELETE FROM `pool_template` WHERE `entry` BETWEEN @OBJECTPOOLS+0 AND @OBJECTPOOLS+;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1696800156876281900.sql:34]: Issue here. Last charater. + what?

File: data/sql/updates/pending_db_world/rev_1696800156876281900.sql [hunk 4]
```diff
@@ -0,0 +1,570 @@
...
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@OBJECTPOOLS+4, 2, 'Durotar Earthroot South Pool 5/5');
+INSERT INTO `pool_gameobject` (`guid`, `pool_entry`, `chance`, `description`) VALUES
+(@OGUID+41, @OBJECTPOOLS+4, 0, 'Durotar Earthroot South 1/5'),
+(@OGUID+42, @OBJECTPOOLS+4, 0, 'Durotar Earthroot South 2/5'),
+(@OGUID+43, @OBJECTPOOLS+4, 0, 'Durotar Earthroot South 3/5'),
+(@OGUID+44, @OBJECTPOOLS+4, 0, 'Durotar Earthroot South 4/5'),
+(@OGUID+45, @OBJECTPOOLS+4, 0, 'Durotar Earthroot South 5/5');
+
+--TODO:  CHECK ALL THESE AND THEN POOL THEM:
```
Comment (heyitsbench) [inline data/sql/updates/pending_db_world/rev_1696800156876281900.sql:194]:
```suggestion
-- TODO:  CHECK ALL THESE AND THEN POOL THEM:
```

May need to add a space here for workflow.

Comment (neifion-00000000) [reply data/sql/updates/pending_db_world/rev_1696800156876281900.sql]: that needed removed entirely actually, it was my note to self from last night

File: data/sql/updates/pending_db_world/rev_1696800156876281900.sql [hunk 5]
```diff
@@ -0,0 +1,568 @@
...
+(@OGUID+1, @OBJECTPOOLS+0, 0, 'Durotar Silverleaf NW 2/8'),
+(@OGUID+2, @OBJECTPOOLS+0, 0, 'Durotar Silverleaf NW 3/8'),
+(@OGUID+3, @OBJECTPOOLS+0, 0, 'Durotar Silverleaf NW 4/8'),
+(@OGUID+4, @OBJECTPOOLS+0, 0, 'Durotar Silverleaf NW 5/8'),
+(@OGUID+5, @OBJECTPOOLS+0, 0, 'Durotar Silverleaf NW 6/8'),
+(@OGUID+6, @OBJECTPOOLS+0, 0, 'Durotar Silverleaf NW 7/8'),
+(@OGUID+7, @OBJECTPOOLS+0, 0, 'Durotar Silverleaf NW 8/8');
+
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@OBJECTPOOLS+1, 3, 'Durotar Silverleaf NW  Pool 2/7');
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/rev_1696800156876281900.sql:487]:
```suggestion
(@OBJECTPOOLS+1, 3, 'Durotar Silverleaf NW Pool 2/7');
```

File: data/sql/updates/pending_db_world/rev_1696800156876281900.sql [hunk 6]
```diff
@@ -0,0 +1,568 @@
...
+INSERT INTO `pool_gameobject` (`guid`, `pool_entry`, `chance`, `description`) VALUES
+(@OGUID+8, @OBJECTPOOLS+1, 0, 'Durotar Silverleaf NW 1/5'),
+(@OGUID+9, @OBJECTPOOLS+1, 0, 'Durotar Silverleaf NW 2/5'),
+(@OGUID+10, @OBJECTPOOLS+1, 0, 'Durotar Silverleaf NW 3/5'),
+(@OGUID+11, @OBJECTPOOLS+1, 0, 'Durotar Silverleaf NW 4/5'),
+(@OGUID+12, @OBJECTPOOLS+1, 0, 'Durotar Silverleaf NW 5/5');
+
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@OBJECTPOOLS+2, 4, 'Durotar Silverleaf West  Pool 3/7');
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/rev_1696800156876281900.sql:496]:
```suggestion
(@OBJECTPOOLS+2, 4, 'Durotar Silverleaf West Pool 3/7');
```

File: data/sql/updates/pending_db_world/rev_1696800156876281900.sql [hunk 7]
```diff
@@ -0,0 +1,568 @@
...
+(@OGUID+18, @OBJECTPOOLS+2, 0, 'Durotar Silverleaf West 6/12'),
+(@OGUID+19, @OBJECTPOOLS+2, 0, 'Durotar Silverleaf West 7/12'),
+(@OGUID+20, @OBJECTPOOLS+2, 0, 'Durotar Silverleaf West 8/12'),
+(@OGUID+21, @OBJECTPOOLS+2, 0, 'Durotar Silverleaf West 9/12'),
+(@OGUID+22, @OBJECTPOOLS+2, 0, 'Durotar Silverleaf West 10/12'),
+(@OGUID+23, @OBJECTPOOLS+2, 0, 'Durotar Silverleaf West 11/12'),
+(@OGUID+67, @OBJECTPOOLS+2, 0, 'Durotar Silverleaf West 12/12');
+
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@OBJECTPOOLS+3, 5, 'Durotar Silverleaf East Valley  Pool 4/7');
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/rev_1696800156876281900.sql:512]:
```suggestion
(@OBJECTPOOLS+3, 5, 'Durotar Silverleaf East Valley Pool 4/7');
```

File: data/sql/updates/pending_db_world/rev_1696800156876281900.sql [hunk 8]
```diff
@@ -0,0 +1,568 @@
...
+(@OGUID+31, @OBJECTPOOLS+3, 0, 'Durotar Silverleaf East Valley 8/14'),
+(@OGUID+32, @OBJECTPOOLS+3, 0, 'Durotar Silverleaf East Valley 9/14'),
+(@OGUID+33, @OBJECTPOOLS+3, 0, 'Durotar Silverleaf East Valley 10/14'),
+(@OGUID+34, @OBJECTPOOLS+3, 0, 'Durotar Silverleaf East Valley 11/14'),
+(@OGUID+35, @OBJECTPOOLS+3, 0, 'Durotar Silverleaf East Valley 12/14'),
+(@OGUID+36, @OBJECTPOOLS+3, 0, 'Durotar Silverleaf East Valley 13/14'),
+(@OGUID+37, @OBJECTPOOLS+3, 0, 'Durotar Silverleaf East Valley 14/14');
+
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@OBJECTPOOLS+4, 8, 'Durotar Silverleaf East Coast  Pool 5/7');
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/rev_1696800156876281900.sql:530]:
```suggestion
(@OBJECTPOOLS+4, 8, 'Durotar Silverleaf East Coast Pool 5/7');
```

File: data/sql/updates/pending_db_world/rev_1696800156876281900.sql [hunk 9]
```diff
@@ -0,0 +1,568 @@
...
+(@OGUID+48, @OBJECTPOOLS+4, 0, 'Durotar Silverleaf East Coast 11/17'),
+(@OGUID+49, @OBJECTPOOLS+4, 0, 'Durotar Silverleaf East Coast 12/17'),
+(@OGUID+50, @OBJECTPOOLS+4, 0, 'Durotar Silverleaf East Coast 13/17'),
+(@OGUID+51, @OBJECTPOOLS+4, 0, 'Durotar Silverleaf East Coast 14/17'),
+(@OGUID+52, @OBJECTPOOLS+4, 0, 'Durotar Silverleaf East Coast 15/17'),
+(@OGUID+53, @OBJECTPOOLS+4, 0, 'Durotar Silverleaf East Coast 16/17'),
+(@OGUID+54, @OBJECTPOOLS+4, 0, 'Durotar Silverleaf East Coast 17/17');
+
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@OBJECTPOOLS+5, 2, 'Durotar Silverleaf Senjin Echo  Pool 6/7');
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/rev_1696800156876281900.sql:551]:
```suggestion
(@OBJECTPOOLS+5, 2, 'Durotar Silverleaf Senjin Echo Pool 6/7');
```

File: data/sql/updates/pending_db_world/rev_1696800156876281900.sql [hunk 10]
```diff
@@ -0,0 +1,568 @@
...
+(@OGUID+55, @OBJECTPOOLS+5, 0, 'Durotar Silverleaf Senjin Echo 1/7'),
+(@OGUID+56, @OBJECTPOOLS+5, 0, 'Durotar Silverleaf Senjin Echo 2/7'),
+(@OGUID+57, @OBJECTPOOLS+5, 0, 'Durotar Silverleaf Senjin Echo 3/7'),
+(@OGUID+58, @OBJECTPOOLS+5, 0, 'Durotar Silverleaf Senjin Echo 4/7'),
+(@OGUID+59, @OBJECTPOOLS+5, 0, 'Durotar Silverleaf Senjin Echo 5/7'),
+(@OGUID+60, @OBJECTPOOLS+5, 0, 'Durotar Silverleaf Senjin Echo 6/7'),
+(@OGUID+61, @OBJECTPOOLS+5, 0, 'Durotar Silverleaf Senjin Echo 7/7');
+
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@OBJECTPOOLS+6, 1, 'Durotar Silverleaf Echo Isles  Pool 7/7');
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/rev_1696800156876281900.sql:562]:
```suggestion
(@OBJECTPOOLS+6, 1, 'Durotar Silverleaf Echo Isles Pool 7/7');
```

**Review by elthehablo — CHANGES_REQUESTED** [review]
nice work

**Review by elthehablo — APPROVED** [review]

**Comment by elthehablo** [comment]
@neifion-00000000 https://github.com/azerothcore/azerothcore-wotlk/pull/17477

**Comment by neifion-00000000** [comment]
> @neifion-00000000 #17477

Despite this not being a universal situation, we have confirmed that you can still see Durotar nodes while phased so I'll fix that asap

**Comment by neifion-00000000** [comment]
as long as there's no complications with the phase mask should be good

File: data/sql/updates/pending_db_world/rev_1696800156876281900.sql [hunk 11]
```diff
@@ -0,0 +1,568 @@
+--
+-- Remove existing Durotar Herbalism Nodes:
+DELETE FROM pool_gameobject WHERE pool_entry BETWEEN 498 AND 501;
```
Comment (heyitsbench) [inline data/sql/updates/pending_db_world/rev_1696800156876281900.sql:3]:
```suggestion
DELETE FROM `pool_gameobject` WHERE `pool_entry` BETWEEN 498 AND 501;
```
Oof, they're gone again.

**Comment by neifion-00000000** [comment]
not my fault :p  my PR was first

**Pull request merged**